common/lib/mandrill_client/client.py

```diff
@@ -38,40 +38,55 @@ class MandrillClient(object):
     REFERRAL_FOLLOW_UP_EMAIL = 'referred-learner-follow-up'
     REFERRAL_SOCIAL_IMPACT_TOOLKIT = 'social-impact-toolkit'
     MINI_COURSE_ENROLMENT = 'mini-course-enrolment'
+    SEND_ACTION_PLAN = 'send-action-plan'
 
     def __init__(self):
         self.mandrill_client = mandrill.Mandrill(settings.MANDRILL_API_KEY)
 
-    def send_mail(self, template_name, user_email, context, attachments=[], subject=None):
+    def attach_emails(self, user_emails):
```

Comment:
- `attach_emails` is little misleading, can we please rename it to `get_email_to` or `get_email_receivers`? or any other intent-full name?
- Also can we please rename the param as well? these are basically `to`, and in context of emails `to` is used for multiple as well. We should also rename it for main send_email method.

Reply:
yes 'to' will be same as it is for multiple emails as mentioned in the mandrill docs.